Source/driver/Castro.cpp

```diff
@@ -1407,12 +1407,19 @@ Castro::initData ()
           auto r = Rad_new[mfi].array();
           auto geomdata = geom.data();
 
+          GpuArray<Real, NGROUPS+1> xnu = {0.0};
+#if NGROUPS > 1
+          ca_get_xnu(xnu.begin());
```

Comment:
Let's avoid the proliferation of this Fortran API please. `xnu` is already available as an array in C++, it's just a class member so we can't pass it directly to the ParallelFor but instead need to make a local copy of it. https://github.com/AMReX-Astro/Castro/blob/main/Source/radiation/MGFLD.cpp#L326-L329

Reply:
okay, I switched this, by making the radiation group info public, and then making a copy before passing into the rad init